cypress/integration/unit_tests/import_data_test.js

```diff
@@ -0,0 +1,216 @@
+import {run} from '../../../docs/import/import_data';
+import {convertEeObjectToPromise} from '../../../docs/map_util';
+import {assertFirestoreMapBounds} from '../../support/firestore_map_bounds';
+import {loadScriptsBeforeForUnitTests} from '../../support/script_loader';
+
+describe('Unit tests for import_data.js', () => {
+  loadScriptsBeforeForUnitTests('ee', 'firebase', 'jquery');
+  let testData;
+  let exportStub;
+  beforeEach(() => {
+    // Create a pretty trivial world: 2 block groups, each a 1x2 vertical
+    // stripes. Under the covers, we scale all dimensions down because
+    // production code creates an "envelope" 1 km wide around damage, and that
+    // envelope is assumed to fully contain any block group that has any damage.
+    const tigerBlockGroups = ee.FeatureCollection(
+        [makeCensusBlockGroup(0), makeCensusBlockGroup(1)]);
+    // Three damage points, one of them outside the blocks, just for fun.
+    // Relevant damage points are in SW and SE blocks.
```

Comment:
what does SW and SE blocks mean?

Reply:
Mostly left over from when I had blocks. Rephrased.